examples/oneview_appliance_device_snmp_v3_users_facts.yml

```diff
@@ -37,11 +37,20 @@
 
     - debug:
         var: appliance_device_snmp_v3_users
-
-    - name: Gather facts about a SNMPv3 user by ID
+    
+    - name: Gather facts about a snmp users by Username
       oneview_appliance_device_snmp_v3_users_facts:
         config: "{{ config }}"
-        id: "2af33d0c-dc1f-4b5f-ba3e-e4a0b1acb899"
+        username: "user123456"
```

Comment:
rename username to name. Because "name" attribute in module params cannot be handled by "set_object_object" method and hence "self.current_resource" will return None even if you provide valid username